### ISSUE TITLE
Feature request: 1 bang, multiple searches

### ISSUE THREAD

**Issue by enrique71** [psidex/CustomBangSearch#3]
Looked at [this FF plugin](https://addons.mozilla.org/en-US/firefox/addon/custom-search-engine/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search) which is too elaborate, however it would be quite useful to be able to define multiple searches such as:
`!d --> `
```
https://wordpress.org/search/%s?in=developer_documentation;https://developer.mozilla.org/en-US/search?q=%s;site:w3schools.com %s
```
which could open in 3 tabs.

**Comment by psidex** [comment]
Implemented with #4, use " :: " to separate your URLs (space, double colon, space)

Sorry this took so long!

:)

**Comment by enrique71** [comment]
Thank you @psidex, lovin' it! 😎
